src/KeyBindingsDefaults.ts

```diff
@@ -77,6 +84,14 @@ const messageComposerBindings = (): KeyBinding<MessageComposerAction>[] => {
                 shiftKey: true,
             },
         },
+        {
+            action: MessageComposerAction.FormatLink,
+            keyCombo: {
+                key: Key.K,
+                ctrlOrCmd: true,
+                altKey: true,
+            },
+        },
```

Comment:
I just tested on macOS and also found that it does not work on Firefox either.
I think <kbd>Ctrl</kbd> + <kbd>Alt</kbd> + <kbd>K</kbd> does not work since it is used for the Δ character on macOS. I am not quite sure why<kbd>Cmd</kbd> + <kbd>Alt</kbd> + <kbd>K</kbd> does not work, but it also does not seem to be a good choice, because under macOS it opens dev tools on Firefox. I think it would be best to switch between <kbd>Shift</kbd> (macOS) and <kbd>Alt</kbd> (All others).  This way the shortcuts also work as expected.